region/maridia/outer.json

```diff
@@ -2505,20 +2647,17 @@
                   "name": "Shinespark",
                   "notable": false,
                   "requires": [
-                    "canSuitlessMaridia",
+                    "h_canNavigateUnderwater",
                     {"canComeInCharged":{
                       "fromNode": 3,
                       "framesRemaining": 0,
                       "shinesparkFrames": 70
                     }}
                   ],
-                  "note": [
-                    "This is suitless only, because with Gravity, another path allows actual access to 5 instead.",
-                    "That path goes 3 -> 7 -> 2 -> 5."
-                  ]
+                  "note": "Shinespark up left to get onto the platform below the door."
```

Comment:
While double checking this commit, I was looking at the wrong thing. I'm not seeing how this shinespark is tricky with gravity suit on. You have plenty of time and just have to hold right to fall onto the platform. What if we changed the frames remaining to something like 5, since sparking through diagonally is not something that the logic can deal with?

Reply:
I was bonking the grapple blocks unless I sparked from the absolute farthest right part of the door.  Having framesRemaining probably fixes this.
Turning off Gravity so you spark less to the left is an interesting interaction.  Wondering if it has a meaningful logic interaction yet aside from adjusting our shinesparkframe costs.

Reply:
Sparking anywhere above the door is fine.  That means this is a problem with building up speed before reaching the door.  With framesRemaining, there is no built up speed so we're fine.